ALI/ALI.py

```diff
@@ -23,15 +23,16 @@
 from CondaSetUp import CondaSetUpCall,CondaSetUpCallWsl
 import time
 import threading
+import sys
 
 
 def check_lib_installed(lib_name, required_version=None):
     try:
-        installed_version = pkg_resources.get_distribution(lib_name).version
+        installed_version =importlib.metadata.version(lib_name)
         if required_version and installed_version != required_version:
             return False
         return True
-    except pkg_resources.DistributionNotFound:
+    except :
```

Comment:
Should check specific error `importlib.metadata.PackageNotFoundError`

Reply:
I made the changement, thank you !